build-constraints.yaml

```diff
@@ -2356,21 +2356,15 @@ skipped-tests:
 
     # Too lazy to keep the test dependencies up to date
     - cases
-    - hasql-postgres
```

Comment:
I think these need to be reverted; the maintainer didn't want to get notified when his test suites were out-of-date. I'll push a commit for that now.